src/OpenTelemetry.AutoInstrumentation.Native/cor_profiler.cpp

```diff
@@ -1200,6 +1220,8 @@ bool CorProfiler::InitThreadSampler()
 
     this->continuousProfiler = new continuous_profiler::ContinuousProfiler();
     this->continuousProfiler->SetGlobalInfo12(this->info12_);
+    this->continuousProfiler->SetGlobalInfo7(this->info_);
+    this->continuousProfiler->SetStackCaptureStrategy(stack_capture_strategy_.get());
```

Comment:
Would it be better separation if we compelety hide `stack_capture_strategy_` inside `continuousProfiler`?